internal/service/project/resource_project.go

```diff
@@ -674,9 +676,15 @@ func SetSlowOperationThresholding(ctx context.Context, performanceAdvisorAPI adm
 	return err
 }
 
-func ReadIsSlowMsThresholdingEnabled(ctx context.Context, api admin.PerformanceAdvisorApi, projectID string) (bool, error) {
+func ReadIsSlowMsThresholdingEnabled(ctx context.Context, api admin.PerformanceAdvisorApi, projectID string, warnings *diag.Diagnostics) (bool, error) {
 	response, err := api.GetManagedSlowMs(ctx, projectID).Execute()
 	if err != nil {
+		if apiError, ok := admin.AsError(err); ok && *apiError.ErrorCode == "USER_UNAUTHORIZED" {
+			if warnings != nil {
+				warnings.AddWarning("user does not have permission to read is_slow_operation_thresholding_enabled", fmt.Sprintf(ErrorProjectRead, projectID, err.Error()))
```

Comment:
```suggestion
				warnings.AddWarning("user does not have permission to read is_slow_operation_thresholding_enabled. Please read our documentation for more information.", fmt.Sprintf(ErrorProjectRead, projectID, err.Error()))
```

Reply:
Doubt here: Should this be a `info` rather than a `warning`? What do you think?

Rationale: my understanding currently is that who tries to **set** this value and does not have permission will get an error (from the `SetSlowOperationThresholding` method), right? 
so effectively this warning appears only for whoever doesn't have permissions in:
- `data source`
- `resource` when they haven't set any value (potentially not interested). 

Imagine someone that upgrades their provider version, don't make any change but start seeing the warning, I'm thinking it's something that can create concerns.

Reply:
In Diagnostics I think only warnings or errors can be added, don't know how we can add info or message with other severity

Reply:
[severity documentation](https://developer.hashicorp.com/terraform/plugin/framework/diagnostics#severity) mentions that warnings should be used for informing `suboptimal situations` and `possible unexpected behaviors`. Maybe we can be more explicit in documentation that no action is required if there is no intention of using the attribute.

Reply:
ok agreed to add to documentation and leave warning. Thank you both!